Tests/GRPCTests/GRPCReflectionServiceTests/reflection.grpc.swift

```diff
@@ -0,0 +1,209 @@
+//
```

Comment:
Can you add a `Generated` directory in `GRPCReflectionServiceTests` and generate the file in there instead? It's more obvious when you're looking through the code where the generated and non-generated code is.